src/plasoscaffolder/bll/mappings/base_mapping_helper.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""Helper methods for mapping"""
+from abc import ABCMeta, abstractmethod
```

Comment:
Also, please only import modules, not the individual classes: http://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting